src/Features/CSharp/Portable/Completion/KeywordRecommenders/RefKeywordRecommender.cs

```diff
@@ -39,9 +40,27 @@ public RefKeywordRecommender()
             };
 
         /// <summary>
-        /// Same as <see cref="SyntaxKindSet.AllGlobalMemberModifiers"/> with ref specific exclusions
+        /// Same as <see cref="SyntaxKindSet.AllGlobalMemberModifiers"/> with ref-specific exclusions
         /// </summary>
         private static readonly ISet<SyntaxKind> RefGlobalMemberModifiers = new HashSet<SyntaxKind>(SyntaxFacts.EqualityComparer)
+            {
+                // SyntaxKind.AsyncKeyword,    // async local functions cannot be byref
+                SyntaxKind.ExternKeyword,
+                SyntaxKind.InternalKeyword,
+                SyntaxKind.NewKeyword,
+                SyntaxKind.OverrideKeyword,
+                SyntaxKind.PublicKeyword,
+                SyntaxKind.PrivateKeyword,
+                SyntaxKind.ReadOnlyKeyword,
+                SyntaxKind.StaticKeyword,
+                SyntaxKind.UnsafeKeyword,
+                SyntaxKind.VolatileKeyword,
```

Comment:
protected is oddly missing.  intentional?

Reply:
i'm guessing intentional since this is top level.  but potentially worth commenting on.

Reply:
`ProtectedKeyword` is not part of `AllGlobalMemberModifiers`. I'm not sure why it's excluded from that set.